frontend/apps/web/src/components/system/Overview.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react'
+import { Model } from '@/model'
+import {
+  CardLayout,
+  ProgressBar,
+  StackLayout,
+  Text,
+  Table,
+  Button,
+} from '@filecoin/ui'
+import { useNavigate } from 'react-router-dom'
+import styled from 'styled-components'
+
+const DATASCORE = {
+  good: 'success',
+  bad: 'error',
+  mediocre: 'warning',
+}
+
+interface Props {
+  model: Model
+  modelName: string
+}
+
+export const Overview: React.FC<Props> = ({ model, modelName }) => {
+  const navigate = useNavigate()
+  const system = model.findSystemByName(modelName)
+  const totalSubsystems = system.subsystems.length
+
+  return (
+    <Wrapper>
+      <ProgressBarWrapper>
+        <ProgressBar
+          data={system.testKindStats.percentages.map(
+            ({ kind, percentage }) => ({
+              name: kind,
+              percentage,
+            }),
+          )}
+          onClick={() => null}
```

Comment:
This is now optional, so we can remove it.

Reply:
done